iphone/Classes/TiUIWebView.m

```diff
@@ -290,7 +290,10 @@ -(void)loadFile:(NSString*)absolutePath
 		textEncodingName:(NSString*)textEncodingName
 	   mimeType:(NSString*)mimeType
 {
-	NSURL *requestURL = [NSURL fileURLWithPath:absolutePath];
+    static unsigned long uniqueIdentifier = 0;
+    uniqueIdentifier++;
```

Comment:
Adding it would be a possible breaking change for some users, that is my only problem. Maybe we could we generate a UUID-identifier in a mutable data structure (like NSSet) every time a local URL is set?